unearthed-arcana/2017/20170206.xml

```diff
@@ -223,5 +223,179 @@
+          <description level="11">
+            <p>As	a	bonus	action,	you	can	grant	an	aegis	to	one	allied	creature	you	can	see	within	60	feet	of	you. Any	bludgeoning,	piercing,	or	slashing	damage	the	target	takes	is	reduced	by	%stone aegis%.	This	effect	lasts	for	1	minute,	until	you	use	it	again,	or	until	you	are	incapacitated.</p>
+            <p>In	addition,	when	a	creature	you	can	see	within	60	feet	of	you	hits	the	protected	target	with	a	melee	attack,	you	can	use	your	reaction	to	teleport	to	an	unoccupied	space	you	can	see	within	5	feet	of	the	attacker.	You	can	teleport	only	if	you	and	the	attacker	are	on	the	same	surface.	You	can	then	make	one	melee	weapon	attack	against	the	attacker.	If	that	attack	hits,	it	deals	an	extra	2d10	force	damage.</p>  
+          </description>
+          <description level="17">
+            <p>As	a	bonus	action,	you	can	grant	an	aegis	to	one	allied	creature	you	can	see	within	60	feet	of	you. Any	bludgeoning,	piercing,	or	slashing	damage	the	target	takes	is	reduced	by	%stone aegis%.	This	effect	lasts	for	1	minute,	until	you	use	it	again,	or	until	you	are	incapacitated.</p>
+            <p>In	addition,	when	a	creature	you	can	see	within	60	feet	of	you	hits	the	protected	target	with	a	melee	attack,	you	can	use	your	reaction	to	teleport	to	an	unoccupied	space	you	can	see	within	5	feet	of	the	attacker.	You	can	teleport	only	if	you	and	the	attacker	are	on	the	same	surface.	You	can	then	make	one	melee	weapon	attack	against	the	attacker.	If	that	attack	hits,	it	deals	an	extra	3d10	force	damage.</p>  
+          </description>
+        </sheet>
+        <rules>
+            <stat name="stones aegis" value="4"  level="6" />
+            <stat name="stones aegis" value="4"  level="8" />
+            <stat name="stones aegis" value="5"  level="10" />
+            <stat name="stones aegis" value="5"  level="12" />
+            <stat name="stones aegis" value="6"  level="14" />
+            <stat name="stones aegis" value="6"  level="16" />
+            <stat name="stones aegis" value="7"  level="18" />
+            <stat name="stones aegis" value="7"  level="20" />
+        </rules>
+      </element>
+
+      <element name="Stone's Edge" type="Archetype Feature" source="Unearthed Arcana" id="ID_WOTC_UA_ARCHETYPE_FEATURE_STONE_SORCERER_STONES_EDGE">
+        <supports />
+        <description>
+          <p>Starting	at	14th	level,	your	mastery	of	earth	magic	allows	you	to	add	the	force	of	elemental	earth	to	your	spells.	When	you	cast	a	spell	that	deals	damage,	choose	one	creature	damaged	by that	spell	on	the	round	you	cast	it.	That	creature	takes	extra	force	damage	equal	to	half	your	sorcerer	level.	This	feature	can	be	used	only	once	per	casting	of	a	spell.</p>
+        </description>
+        <sheet>
+          <description>
+            When	you	cast	a	spell	that	deals	damage,	choose	one	creature	damaged	by that	spell	on	the	round	you	cast	it.	That	creature	takes	extra	%hacky half level% force	damage.
```

Comment:
You can use the **level:sorcerer:half** here, this hacky half level was used when these level statistics were not yet generated by the builder.

Reply:
oh great! thanks